docusaurus/docs/Android/01-basics/01-overview.mdx

```diff
@@ -29,27 +30,42 @@ The client library is a low-level wrapper around the Stream Chat API. It lets yo
 
 It also contains all the basic model objects you'll interact with when using Stream Chat, such as `User`, `Channel`, or `Message`.
 
-### Offline Support
+### State
+
+The state library is a plugin that gives you the possibility to reactively observe data through `StateFlows`.
+Moreover, the plugin adds a sync feature, that for example is responsible for resending messages, reactions, etc, when the user performed actions without an Internet connection.
+
+:::note
+The state plugin doesn't add a persistence layer.<br/> Consider using it together with `Offline Plugin`.
+:::
 
-The offline support is a plugin that adds offline caching capabilities using a local database. For example, it allows you to send messages or add reactions while you're offline. When the user comes back online, the library will automatically recover lost events and retry sending messages.
+<!-- TODO update link once State documentation is ready -->
+This plugin exposes easy-to-use `StateFlow` objects for reading channels, messages, reads, typing, members, watchers and more. See [State](../02-client/06-guides/04-offline-support.mdx) for more details.
```

Comment:
Maybe we should list  `ChannelState` members descriptively here by what they do, such as:

```
This plugin exposes easy-to-use `StateFlow` objects containing information about the channel such as the messages inside it, a list of users currently typing, a list of all channel members and more.
```

The idea is that user reading this is a new user and might not know the nomenclature well.
This is just a suggestion, feel free to reject it if you do not like it.